SeaSharpe_CVGS/SeaSharpe_CVGS/Models/Validation.cs

```diff
@@ -0,0 +1,229 @@
+﻿/*
+ * File Name: Validation.cs
+ * This class contains the validation for the entities 
+ *  
+ * Revision History:
+ *      19-Nov-2015: Created the class, Wrote code, Commented
+ *      
+ * 
+ */
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Web;
+
+namespace SeaSharpe_CVGS.Models
+{
+    public class Validation
+    {
+    }
+
+    /// <summary>
+    /// MetaData Validation for Member
+    /// </summary>
+    [MetadataType(typeof(MemberMetadata))]
+    public partial class Member
+    {
+        class MemberMetadata
+        {
+            [Required(ErrorMessage = "Required")]
+            public virtual ApplicationUser User { get; set; }
+        }
+    }
+
+    /// <summary>
+    /// MetaData Validation for Employee
+    /// </summary>
+    [MetadataType(typeof(EmployeeMetadata))]
+    public partial class Employee
+    {
+        class EmployeeMetadata
+        {
+            [Required(ErrorMessage = "Required")]
+            public virtual ApplicationUser User { get; set; }
+        }
+    }
+
+    /// <summary>
+    /// MetaData Validation for Address
+    /// </summary>
+    [MetadataType(typeof(AddressMetadata))]
+    public partial class Address
+    {
+        class AddressMetadata
+        {
+            [Required(ErrorMessage = "Required")]
+            public virtual Member Member { get; set; }
+
+            [Required(ErrorMessage = "Required")]
+            [StringLength(255, MinimumLength = 1, ErrorMessage = "Maximun lenght is 255 characters and must have at least 1 character")]
```

Comment:
You typo'd lenght instead of length. Also you can use these in an error message:
- {0} - Name
- {1} - Maximum Length
- {2} - Minimum Length

So maybe use `[StringLength(255, MinimumLength = 1, ErrorMessage = "Your {0} needs to be between {1} and {2} characters long.")]`

That would show the user: "Your StreetAddress needs to be between 1 and 255 characters long"

To get it to print "Street Address" as two separate words we can use the attribute [DisplayName("Street Address")] but maybe that belongs in `IdentityModels.cs` instead of `Validation.cs`...